src/components/Passenger Details/PassengerCard.tsx

```
@@ -0,0 +1,268 @@
+    useAppContext();
+
+  // Function to update the passenger name and ensure it has more than 3 characters
+  const handleName = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newName = e.target.value;
+
+    setPassengerDetails((prev) => {
+      const updatedPassengers = [...prev.passengers];
+      updatedPassengers[passengerIndex] = {
+        ...updatedPassengers[passengerIndex],
+        name: newName,
+        validName: newName.length > 3 ? true : false,
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)